best-practices/pd-scheduling-best-practices.md

```diff
@@ -257,10 +257,16 @@ Similar to slow scheduling, the speed of region merge is most likely limited by
 
 - If it is known from metrics that there are a large number of empty regions in the system, you can adjust `max-merge-region-size` and `max-merge-region-keys` to smaller values to speed up the merge. This is because the merge process involves replica migration, so the smaller the region to be merged, the faster the merge is. If the merge operators are already generated rapidly, to further speed up the process, you can set `patrol-region-interval` to `10ms`. This makes region scanning faster at the cost of more CPU consumption.
 
-- A lot of tables have been created and then emptied (including truncated tables). These empty regions cannot be merged if the split table attribute is enabled. You can disable this attribute by adjusting the following parameters:
+- A lot of tables have been created and then emptied (including truncated tables). These empty Regions cannot be merged if the split table attribute is enabled. You can disable this attribute by adjusting the following parameters:
 
     - TiKV: Set `split-region-on-table` to `false`. You cannot modify the parameter dynamically.
-    - PD: Set `key-type` to `"txn"` or `"raw"`. You can modify the parameter dynamically.
+    - PD
+        - Set `key-type` to `"txn"` or `"raw"`. You can modify the parameter dynamically.
+        - Keep `key-type` as `table` and set `enable-cross-table-merge` to `true`. You can modify the parameter dynamically.
+
+    > **Note:**
+    >
+    > After placement rules are enabled, properly switch the value of `key-type` between `txn`and `raw` to avoid the failure of decoding.
```

Comment:
```suggestion
        > **Note:**
        >
        > After placement rules are enabled, properly switch the value of `key-type` between `txn` and `raw` to avoid the failure of decoding.
```